test/util/helpers.ts

```diff
@@ -121,11 +121,12 @@ export function buildMockParamsWithBlockParamAt(
  */
 export function buildMockParamsWithoutBlockParamAt(
```

Comment:
This function is called "build mock params _without_ block param". So if you want to add a block param at a specific spot, does it make sense to use `blockMockParamWithBlockParamAt` instead of modifying this function?

Reply:
I fixed this